sigoptlite/broker.py

```diff
@@ -88,7 +89,12 @@ def create_suggestion(self):
     else:
       source = GPSource(self.experiment)
       self.hyperparameters = source.update_hyperparameters(self.observations, self.hyperparameters)
-    suggestion_data = source.get_suggestion(self.observations)
+
+    # Try and generate a suggestion, otherwise fallback to random search
+    try:
+      suggestion_data = source.get_suggestion(self.observations)
+    except SigoptComputeError:
+      suggestion_data = RandomSearchSource(self.experiment).get_suggestion(self.observations)
```

Comment:
I think what we wanted to do is to catch a specific error, instead of the generic `SigoptComputeError`. If something within libsigopt that breaks, we actually do want that to error out. See the `call_libsigopt_views` implementation.

We can either match the error message for this try/except block or create a subclass like `EmptySuggestionError`.